extensions/default/src/getHangingProtocolModule.js

```diff
@@ -100,8 +100,33 @@ const testProtocol = {
         layoutType: 'grid',
         properties: {
           rows: 1,
-          columns: 1,
-          viewportOptions: [],
+          columns: 2,
+          viewportOptions: [
+            // {
```

Comment:
Why do we have a test protocol in the default hanging protocol module. You should define this in your mode/extension no? We should pull it out so we aren't using baked in stuff for the 3D hanging protocols.